strings/c-or-cpp/Permutation of String.c

```diff
@@ -0,0 +1,57 @@
+#include<stdio.h>
+//declare permutation and swap functions
+void permutation(char *,int,int);
+void swap(char *,char *);
+main()
```

Comment:
Defaulting `main()` to `int main()` has been deprecated in C99 standards, although some compilers still support it but then it will make this code implementation dependent.